src/components/_index.scss

```diff
@@ -5,6 +5,7 @@
 ////
 
 // Application.
+@import 'Breadcrumb/index';
```

Comment:
These were just missing from the component stylesheet? Guessing they were being brought in by other components?

Reply:
We had it further below, but I rearranged it to the top